### ISSUE TITLE
NEO M8N disarmed

### ISSUE THREAD

**Issue by dms-rj** [Xinyuan-LilyGO/LilyGo-LoRa-Series#165]
I would like to receive a step by step method to restore or reset the NEO M8N  of the T BEAM AXP2101 with SX1262 and ESP32-DOWDQ6 V3. I've tried a lot of simple solutions such as simple reset by RST button of the own radio and reset firmware on the flasher.mashtastic.org and I received the following images, the issue was provoqued due to an installation of a display OLED 1.3 with 3 buttons, it cames arrives with an issue that was resolved with a jumper oriented by the Lilygo store, solution that worked in two devices but not on other two (I have four), and did red light up and turned off the GPS function.
![WhatsApp Image 2024-05-24 at 09 49 35](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/170731694/5cd1ac6f-392d-44eb-baa7-a75b1ea77f6a)
![WhatsApp Image 2024-05-24 at 10 05 02](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/170731694/3e46cdf1-01fc-416f-b056-cb410b2630db)
![WhatsApp Image 2024-05-24 at 09 51 10](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/170731694/4022f2d4-ecd5-431e-a42d-e064985e47c9)
![WhatsApp Image 2024-05-24 at 10 01 31](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/170731694/a8d9a66f-cac6-4140-8795-14642ba7df6f)
![WhatsApp Image 2024-05-24 at 10 04 13](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/170731694/e4acd5f1-0d89-4a85-a764-4af2d99445e5)

**Comment by lewisxhe** [comment]
Try the method here to see if the GPS can be reset
https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/tree/master/firmware/GPS_Reset

**Comment by dms-rj** [comment]
> Try the method here to see if the GPS can be reset https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/tree/master/firmware/GPS_Reset

I',ve tried that but dind't work. I guessed that I killed the T-Beam with it procedure (i am a noobie in electronic), so I've tried a solution of the Lilygo engineers and all the T-Beam turned back. They sent me a ESP32 app and a recovery file that I recharged and reseted the T-Beam. Than I recharged the T-Beam firmware version and now I am testing it normaly.
